srcpkgs/drawterm-9front/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'drawterm-9front'
+pkgname=drawterm-9front
+version=r393.cdbacd80f61a
+revision=1
+hostmakedepends="mercurial"
+makedepends="libX11-devel libXt-devel"
+short_desc="Drawterm fork by 9front project"
+maintainer="Juan Cuzmar <juan.cuzmar.s@gmail.com>"
+license="LPL-1.02"
+homepage="http://drawterm.9front.org/"
+CFLAGS="-fno-strict-aliasing"
+
+if [ "${XBPS_TARGET_MACHINE}" = armv5tel ]; then
+	CFLAGS+=" -DARMv5"
+fi
+
+do_fetch() {
+	hg clone https://code.9front.org/hg/drawterm ${wrksrc}
```

Comment:
If upstream doesn't do tags and you have to use a specific commit, you should check it out here. Cloning the source and using that (when it can change from day to day) is a nono.

Reply:
Perfect. I'll grab the last one.

Reply:
I'm thinking about taking the rev option from [mercurial](https://www.mercurial-scm.org/wiki/Revision)
and it would be like this: ```hg clone -r 393 https://code.9front.org/hg/drawterm```
What do you think?

Reply:
I answered this above c: